### PR TITLE
MNT: Update for conda-build 3

### LINKED ISSUE #7: @conda-forge-admin, please update for cb3
Issue:

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've updated the recipe for conda-build 3 as instructed in #7.

List of changes done to the recipe:
Renamed build with host
Moving pkg-config from host to build
Removing pinnings for glib to use values from conda_build_config.yaml. If you need the pin see [here](https://conda-forge.org/docs/meta.html#pinning-packages) for details.
Removing pinnings for xz to use values from conda_build_config.yaml. If you need the pin see [here](https://conda-forge.org/docs/meta.html#pinning-packages) for details.
Adding C++ compiler

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #7

### CODE REVIEW

**Comment by isuruf** [comment]
@scopatz, you should move the previous packages to broken, because they were linking with libstdc++ because it used g++ since `toolchain` wasn't a requirement

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,9 +23,11 @@ build:
 requirements:
   build:
     - pkg-config
+    - {{ compiler('cxx') }}
+  host:
     - libsigcpp
-    - glib 2.51.*
-    - xz 5.2.*
+    - glib
+    - xz
```
Comment (isuruf) [inline recipe/meta.yaml:32]: Aren't these run requirements as well?

File: recipe/build.sh [hunk 2]
```diff
@@ -1,20 +1,8 @@
 #!/usr/bin/env bash
-set -e
+set -ex
 
 export CFLAGS="-O3"
 export CXXFLAGS="-O3"
```
Comment (isuruf) [inline recipe/build.sh]: These should be appended instead of overridden

Comment (scopatz) [reply recipe/build.sh]: Good point!

**Comment by scopatz** [comment]
Finally!

**Pull request merged**